packages/remote-replay-launcher/src/index.ts

```diff
@@ -155,6 +151,33 @@ export const executeRemoteTestRun = async ({
 
   const completedTestRun = await testRunCompleted.promise;
 
+  const tunnelUnlocked = defer<void>();
+  let tunnelCheckInterval: NodeJS.Timeout | undefined = undefined;
+  const checkUnlocked = async () => {
+    const isLocked = await getIsLocked({ client, deploymentId });
+    if (isLocked) {
+      onTunnelLocked?.();
```

Comment:
Does this need to be periodically called? It might be sufficient for this to be called once and let callers implement their own logic for periodic logging etc.

Reply:
Also possible to make it a `onTestRunCompleted?: ({testRun: TestRun, keepingTunnelOpen: boolean}) => void;` or similarly-named callback. If keeping as is I think the naming / docstring of this callback can be improved to make it clearer it's called periodically and after the test run has completed.